### PR TITLE
Update Null terminator position for `arg_str`

### PR DESCRIPTION
**Pull request by NuclearTea**
#61 

Ensured that function without params (hello world), function with integer params (echo back uint32 args) and function with string params work as intended

### CODE REVIEW

**Comment by DeflateAwning** [comment]
Fixes #61

File: firmware/Core/Src/rtos_tasks/rtos_tasks.c [hunk 1]
```diff
@@ -121,9 +125,10 @@ void TASK_handle_uart_telecommands(void *argument) {
 
 			// TODO: maybe log/print args too
 			
-			uint8_t args_str_no_parens[end_of_args_idx - start_of_args_idx + 2];
-			memcpy(args_str_no_parens, &latest_tcmd[start_of_args_idx+1], end_of_args_idx - start_of_args_idx - 1);
-			args_str_no_parens[end_of_args_idx - 1] = '\0';
+			const uint8_t arg_len = end_of_args_idx - start_of_args_idx - 1;
```
Comment (DeflateAwning) [inline firmware/Core/Src/rtos_tasks/rtos_tasks.c:128]: `arg_len` may overflow. `uint8_t` is not long enough.

**Review by DeflateAwning — CHANGES_REQUESTED** [review]
Perfect other than the int size issue

File: firmware/Core/Src/rtos_tasks/rtos_tasks.c (hunk 1 above)
Comment (NuclearTea) [reply firmware/Core/Src/rtos_tasks/rtos_tasks.c]: Made uint16, max value is ~65000, I think we won't have an argument that long 😄

**Review by DeflateAwning — APPROVED** [review]
Looks good to me. Thanks for fixing that! Please squash and merge!

**Pull request merged**